scalalib/src/Lib.scala

```diff
@@ -89,9 +89,16 @@ object Lib{
       // in Scala <= 2.13, the scaladoc tool is included in the compiler
       scalaCompilerIvyDeps(scalaOrganization, scalaVersion)
 
-  def scalaRuntimeIvyDeps(scalaOrganization: String, scalaVersion: String) = Agg[Dep](
-    ivy"$scalaOrganization:scala-library:$scalaVersion".forceVersion()
-  )
+  def scalaRuntimeIvyDeps(scalaOrganization: String, scalaVersion: String) =
+    if (mill.scalalib.api.Util.isDotty(scalaVersion))
+      Agg(
+        // note that dotty-library has a binary version suffix, hence the :: is necessary here
+        ivy"$scalaOrganization::dotty-library:$scalaVersion".forceVersion()
```

Comment:
@smarter, thanks for your explanation. Do you also know why dotty-library uses a binary suffix (_0.24), whereas the scala 2 lib does not?

Reply:
I decided to use a binary suffix for dotty-library so that it would look like any other scala jar. I suspect scala-library doesn't use a suffix just because this convention was invented a long time after scala itself

Reply:
great, thanks. Was just curious :)